### PR TITLE
Another template update

### PR DESCRIPTION
**Pull request by WackerO**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnadeseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/rnadeseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/samplesheet.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sample,fastq_1,fastq_2
```
Comment (ggabernet) [inline assets/samplesheet.csv:1]: This is not updated, please remove or update

File: tests/test_full.yml [hunk 2]
```diff
@@ -4,11 +4,11 @@
     - test_full
   files:
     - path: results_test/differential_gene_expression/DE_genes_tables/DE_contrast_condition_genotype_WT_vs_KO.tsv
-      md5sum: 93fd663cb976c102144e1ffb34d88e53
+      md5sum: b19e34f880a7cb14e031a00d6c8f1b74
```
Comment (ggabernet) [inline tests/test_full.yml:7]: Please just remove the md5sums if they are not kept constant

File: tests/test_star_rsem.yml [hunk 3]
```diff
@@ -4,13 +4,13 @@
     - test_star_rsem
   files:
     - path: results_test/differential_gene_expression/DE_genes_tables/DE_contrast_condition_cellline_H1_vs_GM12878.tsv
-      md5sum: f41854e3d6d69f0c6dc556dfd82b2874
+      md5sum: 96ae6d749099958313b30243f10dfaa1
```
Comment (ggabernet) [inline tests/test_star_rsem.yml:7]: please just remove the md5sums if they are not constant over runs

File: tests/test_full.yml (hunk 2 above)
Comment (WackerO) [reply tests/test_full.yml:7]: Here, the 5sum change was expected. I copied the test.yml and renamed it to test_full.yml, so the old 93f... sum is from test.yml, but the two configs have a different parameter that causes the results to change, so this is fine actually

File: tests/test_star_rsem.yml (hunk 3 above)
Comment (WackerO) [reply tests/test_star_rsem.yml:7]: see above

File: assets/samplesheet.csv (hunk 1 above)
Comment (WackerO) [reply assets/samplesheet.csv:1]: Hmm, what do you mean? I just checked with the [pipeline template](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/assets/samplesheet.csv), to me it seems the samplesheet is up-to-date

Comment (ggabernet) [reply assets/samplesheet.csv:1]: yes of course the template contains an example samplesheet, but this should be adapted to reflect the samplesheet that should be used to run the pipeline. It is an example samplesheet for running the pipeline. As you have already a samplesheet example here: https://github.com/qbic-pipelines/rnadeseq/blob/master/testdata/Sample_preparations.tsv
You can either remove the assets/samplesheet.csv or copy the Sample_preparation.tsv here.

**Review by ggabernet — APPROVED** [review]
Just the samplesheet change and looks good to me

**Pull request merged**